Tests/ComposableArchitectureTests/EffectFailureTests.swift

```diff
@@ -1,58 +1,66 @@
 #if DEBUG
-  import Combine
-  import ComposableArchitecture
-  import XCTest
-
-  @MainActor
-  final class EffectFailureTests: XCTestCase {
-    var cancellables: Set<AnyCancellable> = []
+#if canImport(OpenCombine)
+import OpenCombine
+#else
+import Combine
+#endif
+import ComposableArchitecture
+import XCTest
 
-    func testTaskUnexpectedThrows() async {
-      guard #available(iOS 15, macOS 12, tvOS 15, watchOS 8, *) else { return }
+@MainActor
+final class EffectFailureTests: XCTestCase {
+  var cancellables: Set<AnyCancellable> = []
 
-      var line: UInt!
-      XCTExpectFailure {
-        $0.compactDescription == """
-          An "EffectTask.task" returned from "\(#fileID):\(line+1)" threw an unhandled error. …
+  func testTaskUnexpectedThrows() async throws {
+    guard #available(iOS 15, macOS 12, tvOS 15, watchOS 8, *) else { return }
+    try XCTSkipIfWindowsExpectFailure()
 
-              EffectFailureTests.Unexpected()
+    var line: UInt!
+    XCTExpectFailure {
+      $0.compactDescription
+        == """
```

Comment:
This is a funky change. Intentional?

Reply:
Hmm unsure, it looks _mostly_ like indentation getting screwed up (a lot of the TCA files seemed to have weird indentation) I can fight with this a little bit, but I might just leave it since it's on a more standard indentation pattern now.